src/api/Customer/types/getPaymentDetails.types.ts

```diff
@@ -21,7 +21,8 @@ type PaymentDetailCardGooglePay = GeneralPaymentDetail & {
     | 'android'
     | 'ideal'
     | 'bancontact_card'
-    | 'bancontact_mobile';
+    | 'bancontact_mobile'
+    | 'gcash';
```

Comment:
I think it would be best to have just one instance of the list of the possible payment methods. please reuse the type from `appRedux/types` so we don't need to support two separate lists that describe the same thing :)